content/docs/0.19.x/define/infracost/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Demo: Quality gate evaluation using metrics from another tool
```

Comment:
Hugo is confused about the "Demo:" part and thinks it's another key like "title" or "description". 
This should fix the breaking build.
```suggestion
title: "Demo: Quality gate evaluation using metrics from another tool"
```

Reply:
Done